certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/certificatemodel/model/PdfSpecification.java

```diff
@@ -0,0 +1,17 @@
+package se.inera.intyg.certificateservice.domain.certificatemodel.model;
+
+import java.util.List;
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder
+public class PdfSpecification {
+
+  CertificateType certificateType;
+  int mcid;
+  int signatureWithAddressTagIndex;
```

Comment:
signatureWithAddressTagIndex & signatureWithoutAddressTagIndex could both the records of type TagIndex?